migration/src/androidTest/kotlin/kin/sdk/migration/MigrationManagerIntegrationTest.kt

```diff
@@ -505,7 +509,17 @@ class MigrationManagerIntegrationTest {
         })
         assertTrue(latch.await(timeoutDurationSecondsVeryShort, TimeUnit.SECONDS))
         assertTrue(!migrationDidStart.get())
-        assertTrue(isOldSdk.get())
+
+//        oldKinClient?.let {
```

Comment:
what happened here?
if this code is not needed anymore, delete it...

Reply:
ok